glambie/const/data_groups.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class GlambieDataGroup():
+    name: str
+    long_name: str  # might add some more here later
+
+
+class GlambieDataGroups():
```

Comment:
Think the last comment was lost..(?)

This looks like a readonly dictionary. It probably doesn't need to be readonly, in which case just inherit from dict. If it does need to be readonly, you could follow something like this: 
https://stackoverflow.com/questions/19022868/how-to-make-dictionary-read-only

If you inherit from dict you get this lookup stuff for free